src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -6924,6 +6926,17 @@ export class StudyViewPageStore {
         });
     }
 
+    @computed
```

Comment:
why do we put these in computed (instead of treating them like patientTreatments below as an instance property)?   @Luke-Sikina you might just have copied an existing thing, but i'm curious.

Reply:
I'm not going to resolve this comment just in case it's still relevant, but I think these do need to be computed now, as `this.studyIds` is an observable.

Reply:
yes but they would be computed either way.  i'm just curious why it is we treat these different than we normally treate remoteData, at least in other stores.    probably there isn't an answer.  
normal way (note that this will have same computed behavior):
```
readonly localPro = remotedata(...)
```